docs/notebooks/proton_radiography_particle_tracing.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
<p>It'd be really cool to show radiographs from other example fields here, so that people can see that this doesn't just sample a circle!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I agree (and I need to include more good examples to showcase), but I do have one concern: running the code to create nice pictures for non-trivial examples take non-trivial computational resources, and I wasn't sure we wanted that happening on the back end every time the notebook is updated. Thoughts?

Reply:
In that case we can just run it once and remember not to clear the outputs for this particular notebook. That way, the notebook won't get executed on CI builds. That's exactly why I added nbsphinx, and I'm happy it's finally being useful!

Reply:
Perfect - I've added a couple more examples, then, and I will no longer clear the output so that it doesn't eat up time on CI.